resources/markdown/samReadme.md

```diff
@@ -19,7 +19,7 @@ ${COMPANYNAME} Lambda functions not defined in the [`template.yaml`](./template.
 
 ## Deploying Serverless Applications
 
-You can deploy a serverless application by invoking the `AWS: Deploy SAM application` command through the Command Palette or by right-clicking the Lambda node in the ${COMPANYNAME} Explorer and entering the deployment region, a valid S3 bucket from the region, and the name of a CloudFormation stack to deploy to. You can monitor your deployment's progress through the **${COMPANYNAME} Toolkit** Output Channel.
+You can deploy a serverless application by invoking the `AWS: Deploy SAM application` command through the Command Palette or by right-clicking the Lambda node in the ${COMPANYNAME} Explorer and entering the deployment region, a valid S3 bucket from the region, and the name of a CloudFormation stack to deploy to. You can monitor your deployment's progress through the **${COMPANYNAME} Toolkit\*\* Output Channel.
```

Comment:
prettier keeps getting confused about this. Maybe we can change it to double quotes or backticks?

Reply:
Oh it was prettier doing this...

Yeah I can try something.

Reply:
Neither worked. Weird.
It might have something to do with `**$`?

Reply:
I'd just change the `**` to backticks or something.

Reply:
Backticks looks better anyway.